GREEDY/DataManagers/FileImageGetter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Drawing;
+using System.Windows.Forms;
+
+namespace GREEDY.DataManagers
+{
+    class FileImageGetter : IImageGetter
+    {
+        private readonly OpenFileDialog _getPathDialog;
+
+        public FileImageGetter()
+        {
+            _getPathDialog = new OpenFileDialog
+            {
+                Filter = Environments.AppConfig.FilterImageFiles,
+                FilterIndex = 2,
+                RestoreDirectory = true
+            };
+        }
+
+        public Bitmap GetImage()
+        {
+            if (_getPathDialog.ShowDialog() == DialogResult.OK)
+            {
+                if (_getPathDialog.FileName != "")
+                {
+                    var imageBitmap = new Bitmap(_getPathDialog.FileName);
+                    return imageBitmap;
+                }
+            }
+            // what would a using acoomplish here?
+            return null;
```

Comment:
I suppose that in the file dialog window if you click "cancel" it would return null. Because of that in mainScreen.cs after we define the image in InserFile_Button_Click method, it should be checked if the image is null. Somewhere in the comments i saw Aidas also pointed out some problem about that.

Reply:
Yes, we can implement link to return function to the begin window, but now I just leave it for the future implementation and pointed it as exception. I dont want to finish pur project now, I just change coding style and implementing some useful areas fot the future coding. 😇 after this implementation we will have a possibility to code independently. So, all this exceptions could be my or ours future tasks :)

Reply:
Ok, understood :)